manifests/params.pp

```diff
@@ -8,6 +8,12 @@
 # It sets variables according to platform
 #
 class kafka::params {
+
+  # this is all only tested on Debian and RedHat
+  # params gets included everywhere so we can do the validation here
+  unless $facts['os']['family'] =~ /(RedHat|Debian)/ {
+    fail("${facts['os']['family']} is not supported")
+  }
```

Comment:
We found that blocking other OSs caused issues because sometimes it WOULD work, just wasn't tested. Or it would be close so someone would take it the rest of the way. Is this going to be a standard?